### PR TITLE
Run clippy on nightly

### PR DESCRIPTION
**Pull request by robyoung**
Blocking #5

It seems to make more sense to run against the latest consensus on good practice.

### CODE REVIEW

**Comment by eldruin** [comment]
Is there a fixed version other than nightly which is fine with #5?
Clippy is evolving pretty quickly and I wanted to avoid new improvements there "randomly" breaking the build, which has happened to me in other projects.
I run clippy locally with the last Rust version every time I touch a driver, though.

**Comment by robyoung** [comment]
@eldruin absolutely; I've changed it to `stable`

**Comment by eldruin** [comment]
Sorry for the back and forth, can you rather set 1.53.0 then?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [1.31.0]
+        rust: [stable]
```
Comment (eldruin) [inline .github/workflows/build.yml:95]:
```suggestion
        rust: [1.53.0]
```

**Comment by eldruin** [comment]
Ok I could do this myself. Thanks!

**Pull request merged**